tests/test_envs.py

```diff
@@ -92,21 +92,6 @@ def test_dataset_all():
         logger.debug(f"Expect {supervised_count} envs to support supervised learning")
 
 
-def test_print_all():
```

Comment:
Why is it useless? It checks that all environments in ENVS (maybe using ENVS_NOPSYCHOPY would be better) can be created without crashing and printed (i.e., they can be instantiated and __str__ or __repr__ works). I would keep it.